lib/exercism/user.rb

```diff
@@ -21,6 +21,12 @@ class User < ActiveRecord::Base
 
   before_save do
     self.key ||= Exercism.uuid
+
+    # migrate acls when track_mentor changes
+    if track_mentor_changed?
```

Comment:
Sounds good. Shouldn't take to long. Just curious... why the aversion to callbacks for this? (So I don't make the same mistake at work or on one of my own projects)